reactome_analysis_datasets/tests/test_public_dataset_search.py

```diff
@@ -0,0 +1,17 @@
+import logging
```

Comment:
I guess this file is also not necessary anymore?

Reply:
No not necessary anymore.